extensions/wikia/WikiFactory/DeadWikis/maintenance.php

```diff
@@ -359,6 +359,22 @@ protected function disableWikis( $wikis, &$deleted = array(), &$notDeleted = arr
 				$this->deletedCount++;
 				continue;
 			}
+
+			$isActiveSite = true;
+
+			try {
+				$isActiveSite = $this->isActiveSite($id);
+			} catch ( \Swagger\Client\ApiException $e ) {
+				echo "{od} Failed to get most recent post from site: {$e->getMessage()}\n";
+			}
+
+			if ($isActiveSite) {
```

Comment:
Its intentional that we always hit this conditional, even if the `isActiveSite` call fails.  In this case `$isActiveSite` will be true and we won't soft delete the site.  This ensures that a network problem won't cause a site to be soft deleted.